compiler/ballerina-lang/src/main/java/io/ballerina/projects/Package.java

```diff
@@ -40,10 +40,14 @@ static Package from(Project project, PackageConfig packageConfig) {
         return new Package(packageContext, project);
     }
 
-    PackageContext packageContext() {
+    public PackageContext packageContext() {
```

Comment:
We shouldn't be exposing the packageContext from the API. We should keep this package-private.

Reply:
Made it package private